### DISCUSSION TITLE
Devices not showing

### DISCUSSION THREAD

**Discussion by willi7740** [dgreif/ring#1228] [General]
After installing the plugin on Hoobs, and configuring and linking the account I have no devices showing up. can anyone help?

**Comment by thecbucks** [comment]
I am having the same problem with Homebridge were you able to get this to work? It shows in the log but not in the Home app

**Comment by dgreif** [comment]
I recommend you reach out to HOOBS support for issues like this. If it shows in the HOOBS logs, then HOOBS is failing to properly communicate with HomeKit.
